tests/pipeline/test_pipeline_algo.py

```diff
@@ -344,6 +349,72 @@ def handle_data(context, data):
         # Run for a week in the middle of our data.
         algo.run(self.data_portal)
 
+    def test_multiple_pipelines(self):
+        """
+        Test that we can attach multiple pipelines and access the correct
+        output based on the pipeline name.
+        """
+        def initialize(context):
+            pipeline_close = attach_pipeline(Pipeline(), 'test_close')
+            pipeline_volume = attach_pipeline(Pipeline(), 'test_volume')
+
+            pipeline_close.add(USEquityPricing.close.latest, 'close')
+            pipeline_volume.add(USEquityPricing.volume.latest, 'volume')
+
+        def handle_data(context, data):
+            closes = pipeline_output('test_close')
+            volumes = pipeline_output('test_volume')
+            date = get_datetime().normalize()
+            for asset in self.assets:
+                # Assets should appear iff they exist today and yesterday.
+                exists_today = self.exists(date, asset)
+                existed_yesterday = self.exists(date - self.trading_day, asset)
+                if exists_today and existed_yesterday:
+                    self.assertEqual(
+                        closes.loc[asset, 'close'],
+                        self.expected_close(date, asset)
+                    )
+                    self.assertEqual(
+                        volumes.loc[asset, 'close'],
```

Comment:
I think this column name is a typo.  I'm surprised that this test currently passes.